packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,153 @@
+
+For styling:
+
+The accordion has 5 css parts from FAST's API:
+
+-   heading (Wraps the Button)
+-   button (The button which serves to invoke the item)
+-   heading-content (wraps the slot for the heading content within the button)
+-   icon (The icon container)
+-   region (The wrapper for the accordion item content)
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` buttons. The nimble-button pattern with these styles seems to fit the styling of the accordion, so these styles can be used in the heading css class. The attributes of `block`, `outline`, and `ghost` will most likely be implemented with the name `appearance`. A few styling elements like the background color change on click that is used in the button will have to be removed. An arrow pointing to the right would be included in the `start` slot of the button to signify that the accordion is closed.
+
+The Figma design also includes a state that is most likely `error`, as it has a red border and red exclamation mark icon. For this state, the styling of the accordion header button could be altered to have a red border. This could be controlled with an attribute such as `severity`, which would have a default state of `default` and would not require a css class. Adding the error icon could be done by putting the icon in the `end` slot of the accordion heading.
+
+Once the accordion is open, the arrow pointing right would be replaced with an arrow pointing down in the `start` slot to signify that the accordion is open. A css class for the nimble-accordion-item can be used to get a green border (or red for the `error` state) around the entire accordion (parent and children of accordion).
+
+Extra styling would be required to place Nimble components at the start and end of inner accordion items, as well as to center them.
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_ Unchanged
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Special states of the accordion (`error`) and a div with children for Nimble components within inner accordion items
+-   _Slots:_ Unchanged
+
+### Angular integration
+
+An Angular directive will be created for this component. The component will not have form association, so a `ControlValueAccessor` will not be created.
+
+### Blazor integration
+
+A blazor wrapper will be created for the component.
+
+### Additional requirements
+
+-   _User interaction: Do the FAST component's behaviors match the visual design spec? When they differ, which approach is preferable and why?_
+    -   Partially, extra functionality for Nimble components in inner accordion items will be required. Adding this functionality through the `content` section of the accordion will be preferrable because it doesn't require deviating from FAST's API.
```

Comment:
What "extra functionality" is required? I'm not sure I understand.

Reply:
That was mentioning the implementation of divs for component functionality in the accordion children but based on your [comment above](https://github.com/ni/nimble/pull/1371#discussion_r1269942724) it's most likely unnecessary, so I'll change this to No Additional Requirements.